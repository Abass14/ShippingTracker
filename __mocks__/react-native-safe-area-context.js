const inset = {
    top: 0,
    right: 0,
    bottom: 30,
    left: 0,
}

export const SafeAreaProvider = ({ children }) => children

export const SafeAreaConsumer = ({ children }) => children(inset)

export const useSafeAreaInsets = () => inset