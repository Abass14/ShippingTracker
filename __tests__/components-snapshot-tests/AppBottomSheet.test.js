import ShallowRenderer from 'react-shallow-renderer';
import AppBottomSheet from '../../src/app/components/bottomsheet';
import { SafeAreaProvider } from 'react-native-safe-area-context';

it('should render AppBottomSheet correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
        <SafeAreaProvider>
            <AppBottomSheet />
        </SafeAreaProvider>
    )
    expect(tree).toMatchSnapshot()
})