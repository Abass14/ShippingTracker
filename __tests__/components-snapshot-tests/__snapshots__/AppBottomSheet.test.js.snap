// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AppBottomSheet correctly 1`] = `<AppBottomSheet />`;
